hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineScrubber.java

```diff
@@ -57,6 +57,7 @@ public class BackgroundPipelineScrubber implements SCMService {
   private final long intervalInMillis;
   private final long waitTimeInMillis;
   private long lastTimeToBeReadyInMillis = 0;
+  private volatile boolean runImmediately = false;
```

Comment:
Oh, it is a nice trick, so now the UT covers both the running and wait state of the scrubber thread.
Thanks a lot, LGTM.